lib/tesla/adapter/httpc.ex

```diff
@@ -100,4 +100,9 @@ defmodule Tesla.Adapter.Httpc do
 
   defp handle({:error, {:failed_connect, _}}), do: {:error, :econnrefused}
   defp handle(response), do: response
+
+  # it is reasonable to make a post request with an empty body,
+  # however httpc does not natively support it
+  defp format_request_body(%{method: :post} = env), do: env.body || ""
```

Comment:
Does it only apply to POST requests?

Reply:
Honestly I'm not sure about that.

When I first opened the issue, I ran into this problem with some Auth0 POST requests.
I can't really exclude other actions to have the same problem, so I was thinking about maybe starting here and see if this happens in other situations?
Or if you have other preferences I can explore these.